planner/core/logical_plans.go

```diff
@@ -478,7 +482,8 @@ func (ds *DataSource) getHandleCol() *expression.Column {
 // deriveIndexPathStats will fulfill the information that the accessPath need.
 // And it will check whether this index is full matched by point query. We will use this check to
 // determine whether we remove other paths or not.
-func (ds *DataSource) deriveIndexPathStats(path *accessPath) (bool, error) {
+// considerConditions is the conditions used to generate the DetachRangeResult for path.
```

Comment:
What is `considerConditions`? Do you mean `conds`?

Reply:
yes